corehq/ex-submodules/couchforms/const.py

```diff
@@ -18,17 +18,33 @@
                   TAG_NAME, TAG_META, ATTACHMENT_NAME, 'case', MAGIC_PROPERTY]
 
 
+SUPPORTED_MEDIA_FILE_EXTENSIONS = [
+    "jpg", "jpeg", "3gpp", "3gp", "3ga", "3g2", "mp3", "wav", "amr",
+    "mp4", "3gp2", "mpg4", "mpeg4", "m4v", "mpg", "mpeg", "qcp", "ogg"
+]
```

Comment:
where does this list come from?

Reply:
this list was provided on the ticket by @shubham1g5 for the extensions that mobile supports.

quoting more from him
> all those extensions are media files corresponding to 3 media question types we have on HQ - image capture, audio capture and video capture.
there is no text capture question since that would directly be included in the form xml